benchopt/utils/terminal_output.py

```diff
@@ -2,15 +2,16 @@
 import shutil
 import ctypes
 import platform
+import sys
 
 from ..config import DEBUG
 
 
 MIN_LINE_LENGTH = 20
 BLACK, RED, GREEN, YELLOW, BLUE, MAGENTA, CYAN, WHITE = range(30, 38)
 
-CROSS = u'\u2717'
-TICK = u'\u2713'
+CROSS = '\u2717' if sys.platform != 'win32' else 'X'
+TICK = '\u2713' if sys.platform != 'win32' else 'V'
```

Comment:
The support of unicode in the windows shell is not great, and I understand the need to have different cases here.

Just looking at some encoding solutions, maybe a good solution would be to set this:
```python
import sys
sys.stdout.reconfigure(encoding='utf-8')
```

Could you check it?

Reply:
It worked ! Thanks @tomMoral

Reply:
:tada: !!!